src/memray/reporters/templates/base.html

```diff
@@ -139,8 +139,11 @@ <h5 class="modal-title" id="helpModalLabel">How to interpret {{ kind }} reports<
   <script type="text/javascript">
     const packed_data = {{ data|tojson }};
     var data = null;
+    var flamegraphData = null;
+    var invertedImportSystemData = null;
```

Comment:
As above, this makes it sound like this one _does_ include import system nodes, rather than does not. I'd call it something like `importSystemHiddenFlamegraphData` or `noImportSystemData` or `flamegraphDataNoImportSystem` or something.